### PR TITLE
Support quotes for the `--cluster-name` flag

### LINKED ISSUE #799: Using --cluster-name stops any responses
## Description
```
BotKube version: v0.14.0
K8s Server Version: v1.23.10-eks-15b7512
Slack
```

We have multiple clusters running botkube, as as advised by the bot, we try to do commands in this manner:
`@botkube get pods --cluster-name=“myclustername”`
This results in no response.

If I do `@botkube get pods`, all clusters respond with the correct answer.

## Expected behavior

I should have a working mechanism to choose a call/response to a single botkube instance.

### PR DESCRIPTION
**Pull request by pkosiec**
## Description

Changes proposed in this pull request:

- Support quotes for the `--cluster-name` flag
- Suggest in help providing cluster name without quotes (as they might be problematic in some communication platforms - e.g. replaced with different characters)

## Screenshots

Screenshot 2022-10-07 at 14 05 10

image

## Related issue(s)

Resolves #799

### CODE REVIEW

File: pkg/utils/utils.go [hunk 1]
```diff
@@ -122,7 +123,13 @@ func GetClusterNameFromKubectlCmd(cmd string) string {
 	if len(matchedArray) >= 2 {
 		s = matchedArray[1]
 	}
-	return s
+
+	str, err := strconv.Unquote(s)
+	if err != nil {
```
Comment (ezodude) [inline pkg/utils/utils.go:128]: Had to think about this, works.

**Review by ezodude — APPROVED** [review]
👍

File: pkg/utils/utils.go (hunk 1 above)
Comment (pkosiec) [reply pkg/utils/utils.go:128]: if you think I should put a comment or something, or you have better alternative, let me know 👍

Comment (ezodude) [reply pkg/utils/utils.go:128]: Nope all good.

**Pull request merged**